entries/jQuery.htmlPrefilter.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.htmlPrefilter" return="String">
+  <title>jQuery.htmlPrefilter()</title>
+  <desc>Modify and filter HTML strings passed through <a href="/category/manipulation/">jQuery manipulation methods</a>.</desc>
+  <signature>
+    <added>1.12/2.2</added>
+    <argument name="html" type="String">
+      <desc>The html string on which to operate.</desc>
+    </argument>
+  </signature>
+  <longdesc>
+    <p>This method rarely needs to be called directly. Instead, use it as an entry point to modify existing <a href="/category/manipulation/">jQuery manipulation methods</a>. For instance, to remove all scripts from incoming html strings, do this:</p>
+
```

Comment:
Typically our files don't have empty lines but it isn't a big deal and doesn't cause any error.